Final Project Submission/Black Jack.cpp

```diff
@@ -0,0 +1,632 @@
+         << "\nthe total is 17 or more, at which point the dealer must stand. If the"
+         << "\ndealer has an ace, and counting it as 11 would bring his total to 17 or"
+         << "\nmore (but not over 21), he must count the ace as 11 and stand. The dealer's"
+         << "\nare in this way pre-ordained on all plays, whereas the player always the"
+         << "\noption of taking one or more additional cards. Once the dealer's turn is"
+         << "\nfinished, his hand is compared the player's to determine a winner." << endl;
+  }
+
+  if (a==6||a==7)
+  {
+    cout << "\nOther Rules:" << endl
+         << "\tIf a player is delt some face card AND an ace, this is called"
+         << "\nBlackjack, and the player wins automatically. If the dealer has Blackjack"
+         << "\nand the player does not, the dealer wins by defult, even if the player"
+         << "\nhas 21." << endl;
+
+    cout << "\n\tIf a player's first two cards are of the same denomination,";
+    cout << "\nfor example two jacks or two sixes, the player can choose to 'split' his"
+         << "\nhand and play future dealt cards on either one. A bet equal to the original"
+         << "\nbet is placed on each card, and the player continues a chance to win or"
+         << "\nlose on each." << endl;
+
+    cout << "\n\tAnother option open to the player is doubling his bet when the"
+         << "\noriginal two cards dealt total 9, 10, or 11. When the player's turn"
+         << "\ncomes, he places a bet equal to the original bet, and the dealer"
+         << "\ngives him just one card, which is placed face down and is not turned"
+         << "\nup until the bets are settled at the end of the hand. With two fives,"
+         << "\nthe player may split a pair, double down, or just play the hand in the"
+         << "\nregular way. Note that the dealer does not have the option of splitting"
+         << "\nor doubling down." << endl;
+  }
+
+  cout << "\nPLEASE NOTE: due to the somewhat limited scope of this program,"
+       << "\nthis game will NOT be played with the Split or Double Down options offered"
+       << "\nto the user, and aces will always be counted as 11s." << endl;
+}
+
+// geneates and returns first card out of 52
+int getPlayerCard_1()
+{
+  int playerCard_1 = rand()%52 +1;
+  return playerCard_1;
+}
+
+// generates and returns next card out of 51 remaining possibilities
+int getDealerCard_1(int playerCard_1)
+{
+  int dealerCard_1 = rand()%52 +1;
+  int exclude = playerCard_1;
+
+  while (dealerCard_1 == exclude)
+    dealerCard_1 = rand()%52 +1;
+
+  return dealerCard_1;
+}
+
+// generates and returns next card out of 50 remaining possibilities
+int getPlayerCard_2 (int playerCard_1, int dealerCard_1)
+{
+  int exclude_1 = playerCard_1;
+  int exclude_2 = dealerCard_1;
+
+  int playerCard_2 = rand()%52 +1;
+
+  while (playerCard_2 == exclude_1 || playerCard_2 == exclude_2)
```

Comment:
I'd probably code this as `do`/`while` since you know you always want to do the random-generator at least once